prisma/schema.prisma

```diff
@@ -36,6 +35,6 @@ model JournalEntry {
   probiotics        Boolean
   proteinIntake     Int
   authorId          String
-  author            User        @relation(fields: [authorId], references: [id])
+  author            User        @relation(fields: [authorId], references: [id], onDelete: Cascade)
```

Comment:
Allows for the delete to cascade to the journal entries, prisma defaults to not allow this